sample/strategy.py

```
@@ -100,23 +128,49 @@ def strategy(exit_option: ExitOption, buffer: float, predict_file: str) -> List[
+                sum(
+                    trade.entry_size
+                    for trade in trade_list
+                    if (not trade.is_exit() or trade.exit_ts > ts)
+                )
+                + 1
+            )
             if ts in monthly_last:
-                entry_main = f'{m2}'
+                entry_main = f"{m2}"
                 entry_price = settle_dict[m2].loc[ts]
                 assert pd.notna(entry_price)
```

Ruff (0.12.2)

Use of `assert` detected

(S101)